eventually/vote/models.py

```diff
@@ -0,0 +1,210 @@
+"""test"""
+from django.db import models
+
+
+class Vote(models.Model):
+    '''
+        Create Vote model
+
+        Args:
+            * is_active (BooleanField): active vote or end
```

Comment:
add `:param` and `type` directives for describing of class and functions attributes and parameters.